lib/ui/views/home_view.dart

```diff
@@ -22,19 +23,33 @@ class HomeView extends StatelessWidget {
             transitionBetweenRoutes: false,
           ),
         ),
-        body: _Posts(),
+        body: _Body(),
```

Comment:
This build method is too short now, replace _Body with the stateResponsive widget